package.json

```diff
@@ -21,8 +21,9 @@
     "@cypress/webpack-preprocessor": "^6.0.0",
     "@openshift-console/dynamic-plugin-sdk": "^1.0.0",
     "@openshift-console/dynamic-plugin-sdk-webpack": "^1.0.0",
-    "@patternfly/react-core": "^5.1.1",
-    "@patternfly/react-table": "^5.1.1",
+    "@patternfly/react-core": "^6.1.1-prerelease.1",
```

Comment:
Should be changed to same versions as console as part of https://issues.redhat.com/browse/CONSOLE-4464